pkg/operator/operator.go

```diff
@@ -217,6 +218,14 @@ func New(config operatorconfig.Config, kubeConfig *rest.Config) (*Operator, erro
 		return nil, fmt.Errorf("failed to create configurable-route controller: %v", err)
 	}
 
+	// The status controller needs to be aware of conflicting OLM subscriptions in any namespace. In order to prevent
+	// ballooning the main cache to watch all namespaces on all its informers, create a separate cache for tracking OLM
+	// subscriptions across all namespaces.
```

Comment:
Why do we need to track subscriptions across all namespaces? Aren't there a limited number of namespaces that subscriptions should be installed to?  Or are we just being prepared for anything here?

Reply:
It's fine to be prepared for anything.